### DISCUSSION TITLE
Making Updates to the app without the need to re-deploy it

### DISCUSSION THREAD

**Discussion by mnobeidat13** [flet-dev/flet#599] [Q&A]
I am building a portfolio website to show my work. I have a page that previews my projects and I would like to have the option to add new projects without updating my code. 
My projects page looks like this
![image](https://user-images.githubusercontent.com/32016587/202866266-4bc773fe-ecbc-4f20-8e02-6f5260032676.png)
What I need is an option to add a new project but it must be available only for me as an admin.
Is there a way to achieve that?

I am thinking of creating another page which has that option that I can access by providing a key.

**Comment by plugeit** [comment]
bro how you deploy it can you please tell

**Reply by plugeit** [reply]
i can't find any video how to deploy flet app

**Reply by mnobeidat13** [reply]
Instructions here are very clear and straightforward 
https://flet.dev/docs/guides/python/deploying-web-app/replit

**Reply by plugeit** [reply]
thanks bro it worked
